devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy

```diff
@@ -225,6 +234,19 @@ def refresh() {
 	delayBetween(commands,100)
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configure() {
+	unschedule()
+	runEvery5Minutes(healthPoll)
```

Comment:
Z-Wave switches and dimmers are polled as needed from the hub and should **not** be polled regularly from the cloud. Although they have the Polling capability, the polling job is set to exclude them because it causes big problems for large and unhealthy Z-Wave networks. We should be relying on the ping() method to check that these devices are online.

The spec of the hub-based polling system is here: https://smartthings.atlassian.net/wiki/display/HUBPLT/Z-Wave+and+SmartThings#Z-WaveandSmartThings-Hub-BasedSwitchPolling